### ISSUE TITLE
[B7] Cheat sheet is missing from creative mode

### ISSUE THREAD

**Issue by gerbilfat** [Azanor/thaumcraft-beta#368]
I set toggled the option in mod settings to enable the cheat sheet in creative mode for testing purposes. However, the only thaumnomicon available in the creative inventory is the standard one, not the cheat sheet. It's impossible to do testing in creative without having the cheat sheet available. Sorry, I'm not sure if this is a bug or if this is intentional so we don't see spoilers before you want us to. :)

TC6 B7 and Baubles, no other mods installed, single player creative mode. Thank you!

**Comment by 755437985** [comment]
The cheat sheet doesn't work for me as well.
If you wants all the researches, I think you could also use the command "/tc research @p all"

**Comment by Azanor** [comment]
You will need to restart your client after enabling that setting.

**Comment by gerbilfat** [comment]
I enabled the setting, closed out the client completely, and restarted by rolling a brand new creative world. Still no cheat sheet available in the inventory, just the standard Thaumnomicon. I thought this was kind of weird, so I took another look at the mod settings - the cheat sheet setting itself had somehow gotten unflagged when the client was restarted. I tried this a few more times of toggling the setting to true, restarting the client, and checking the mod config page. This is happening with other config settings, too. Something about restarting the client is changing the config preferences back to default.

I'm using the standard MC launcher, no special tools, no other mods besides TC and Baubles, Win10.
